installer/create_installer.sh

```diff
@@ -32,7 +27,7 @@ if ! is_bin_in_path python && is_bin_in_path python3; then
     }
 fi
 
-if [[ -v "VIRTUAL_ENV" ]]; then
+if [ -n "${VIRTUAL_ENV+set}" ]; then
```

Comment:
`-v` doesn't work on some version of bash (like the one installed on my mac, apparently). This is a more generic way to do the same thing.